src/fury/util/buffer.cc

```diff
@@ -58,7 +58,7 @@ Buffer &Buffer::operator=(Buffer &&buffer) noexcept {
 
 Buffer::~Buffer() {
   if (own_data_) {
-    delete data_;
+    free(data_);
```

Comment:
We need to use `free` since it is allocated from `malloc`

Reply:
Good catch